02_classify_mutations/nbconverted/compare_filtering.py

```diff
@@ -0,0 +1,300 @@
+# and that we have data for two replicates (two random seeds)
+print(new_results_df.shape)
+print(new_results_df.seed.unique())
+print(new_results_df.training_data.unique())
+new_results_df.head()
+
+
+# In[4]:
+
+
+# load compressed data for me_27k and me_450k
+new_compressed_results_df = au.load_compressed_prediction_results(
+    new_results_dir, 'gene')
+
+# make sure that we're correctly pointing to compressed methylation data
+# and that we have data for one dimension and two replicates (two random seeds)
+print(new_compressed_results_df.shape)
+print(new_compressed_results_df.seed.unique())
+print(new_compressed_results_df.training_data.unique())
+print(new_compressed_results_df.n_dims.unique())
+new_compressed_results_df.head()
+
+
+# In[5]:
+
+
+new_results_df['n_dims'] = 'raw'
+new_results_df = pd.concat((new_results_df, new_compressed_results_df))
+print(new_results_df.seed.unique())
+print(new_results_df.training_data.unique())
+print(new_results_df.n_dims.unique())
+print(new_results_df.shape)
+training_data_map = {
+    'expression': 'gene expression',
+    'me_27k': '27k methylation',
+    'me_450k': '450k methylation',
+    'rppa': 'RPPA',
+    'mirna': 'microRNA',
+    'mut_sigs': 'mutational signatures',
+}
+new_results_df.training_data.replace(to_replace=training_data_map, inplace=True)
+new_results_df.head()
+
+
+# In[6]:
+
+
+new_all_results_df = au.compare_all_data_types(new_results_df,
+                                               SIG_ALPHA,
+                                               metric='aupr')
+
+new_all_results_df.sort_values(by='p_value').head(10)
+
+
+# In[7]:
+
+
+sns.set({'figure.figsize': (22, 5)})
+sns.set_style('whitegrid')
+
+fig, axarr = plt.subplots(1, 3)
+
+# plot mutation prediction from expression, in a volcano-like plot
+datasets = ['gene expression', '27k methylation', '450k methylation']
+filtered_data_map = {k: v for k, v in training_data_map.items() if v in datasets}
+
+plu.plot_volcano_baseline(new_all_results_df,
+                          axarr,
+                          filtered_data_map,
+                          SIG_ALPHA,
+                          metric='aupr',
+                          verbose=True)
+
+
+# In[8]:
+
+
+sns.set({'figure.figsize': (22, 5)})
+sns.set_style('whitegrid')
+
+fig, axarr = plt.subplots(1, 3)
+
+# plot mutation prediction from expression, in a volcano-like plot
+datasets = ['RPPA', 'microRNA', 'mutational signatures']
+filtered_data_map = {k: v for k, v in training_data_map.items() if v in datasets}
+
+plu.plot_volcano_baseline(new_all_results_df,
+                          axarr,
+                          filtered_data_map,
+                          SIG_ALPHA,
+                          metric='aupr',
+                          verbose=True)
+
+
+# ### Compare against results with old filtering scheme
+
+# In[9]:
+
+
+# load raw data
+old_results_df = au.load_stratified_prediction_results(orig_results_dir, 'gene')
```

Comment:
Is it worth it to change the names from "old" and "new" to something like "total_filtering" vs. "cancerwise_filtering"? If someone in the future is reading this code doesn't know what order you did things in, this notation might be confusing. But it's not too hard to figure out, so your call.